index.html

```diff
@@ -16,16 +18,119 @@
 
 <body>
 
-    <div class="container">
-      <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+  <div class="container">
+
+    <div class="row">
+      <div class="col-md-12">
+        <nav class="nav">
+          <a href="#" class="nav-link">Bio</a>
+          <a href="#" class="nav-link">Work History</a>
+          <a href="#" class="nav-link">Skills</a>
+          <a href="#" class="nav-link">Education</a>
+        </nav>
+      </div>
+    </div>
+    <div class="top-content">
+      <div class="row jumbo">
+        <div class="col-sm-12 col-md-4 col-xl-3 imageHere">
+          <img src="img/project-pic.jpg">
+        </div>
+        <div class="col-md-8 top-text">
+          <h1>A Resume, but Digital?</h1>
+          <p> Why yes! I thought this might actually be a more apt way to portray my skills to you - possible future employer.
+            <p>My name is <B>Cruise Brantley</B> and I have various qualities that I believe would be an asset to your company, including
+              building this resume! I hope you'll take my application into serious consideration.</p>
+          </p>
+        </div>
+      </div>
+      <div class="row cardsGroup">
+        <div class="col-4 cards">
+          <div class="card" style="width: 18rem;">
+            <img class="card-img-top lambda" src="img/lambda.png" alt="Card image cap">
+            <div class="card-body">
+              <h5 class="card-title">Lambda School</h5>
+              <p class="card-text">Lambda School has taught me everything I know about Web Development!</p>
+            </div>
           </div>
+        </div>
+        <div class="col-4">
+          <div class="card" style="width: 18rem;">
```

Comment:
I'm assuming you put `"style="width: 18rem;"` because you wanted to make your cards smaller. In doing so, this altered the alignment of your cards. If you take a closer look at your project, you will see that the cards have aligned itself to the left. A fast way to fix this is by taking out the style completely. Or you can make the cards smaller but also align in the center by possibly adding margins to the side in your css. But when I tried this, it looked a little wonky. Maybe you will have better luck, haha.

The purpose of bootstrap is to have things automatically aligned for you which is also it's downfall in some cases!